cmd/iface.js

```diff
@@ -60,19 +60,20 @@ function indicator (value, type = 'success') {
 }
 
 const outputter = (cmd, taggers = {}) => async (json, stream, state = {}) => {
+  let error = null
   try {
     for await (const { tag, data = {} } of stream) {
       if (json) {
         print(JSON.stringify({ cmd, tag, data }))
         continue
       }
-      if (tag === 'error') {
-        const err = new Error(data.message)
-        err.sidecarCode = data.code
-        err.sidecarStack = data.stack
-        throw err
+      let result = null
```

Comment:
have you checked that this change doesn't cause breakage for other command outputs? it definitely will if any of them push objects with tag: 'error' 

what's the reason for this change?

Reply:
sorry, that was unintended, let me revert